go/src/koding/kites/kloud/provider/google/stack.go

```diff
@@ -254,6 +257,10 @@ func (s *Stack) ApplyTemplate(c *stack.Credential) (*stack.Template, error) {
 
 	t.Resource["google_compute_instance"] = resource.GCInstance
 
+	t.Variable["koding_stack_id"] = map[string]interface{}{
```

Comment:
Eg.
```yaml
# Here is your stack preview
# You can make advanced changes like modifying your VM,
# installing packages, and running shell commands.
provider:
  google:
    credentials: '${var.google_credentials}'
    project: '${var.google_project}'
    region: '${var.google_region}'
resource:
  google_compute_address:
    google_ca:
      name: 'user-defined-name'
  google_compute_instance:
    # this is the name of your VM
    google-instance:
      # and this is its identifier (required)
      name: google-instance
      # select your instance_type here: eg. n1-standard-1
      machine_type: f1-micro
      # base image for your instance
      disk:
        image: ubuntu-1404-lts
      # select your instance zone which must be in provided region: eg. us-central1-a
      zone: us-central1-a
```
We need to replace "google_ca.name" as well. Because it will fail as already existing resource during re-initialization.